cpp/program/setup.cpp

```diff
@@ -459,6 +459,9 @@ vector<SearchParams> Setup::loadParams(
     if(cfg.contains("rootNumSymmetriesToSample"+idxStr)) params.rootNumSymmetriesToSample = cfg.getInt("rootNumSymmetriesToSample"+idxStr, 1, 16);
     else if(cfg.contains("rootNumSymmetriesToSample"))   params.rootNumSymmetriesToSample = cfg.getInt("rootNumSymmetriesToSample",        1, 16);
     else                                                 params.rootNumSymmetriesToSample = 1;
+    if(cfg.contains("nodeNumSymmetriesToSample"+idxStr)) params.nodeNumSymmetriesToSample = cfg.getInt("nodeNumSymmetriesToSample"+idxStr, 1, 16);
```

Comment:
so 16 here, but 8 below?

Reply:
I just copy/pasted the existing code without thinking about it any. @lightvector, should these two values getInt() with a max value of 8 instead of 16?

Reply:
A higher upper bound makes some sense for with replacements, but yes it seems it should be NNInputs::NUM_SYMMETRY_COMBINATIONS here

Reply:
Right, with replacement I could see pulling more than 8. If we are getting rid of replacement, then we should max at NNInputs::NUM_SYMMETRY_COMBINATIONS.